Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM rust:1.56.1-bullseye AS builder
+FROM rust:1.58.1-bullseye AS builder
```

Comment:
Could I request a bump on the `lcli` image as well :pray:

Reply:
Done. I also bumped the Antithesis image, lemme know if that wasn't the right thing to do